ROADMAP.md

```
@@ -798,14 +798,17 @@ Instead of manually implementing each provider, use LiteLLM to get instant suppo
 **Priority**: Medium
 
 ### 24. Test Parallelism
-**Goal**: Fix test parallelism issues (mainly in terminal_tools)
-**Priority**: High
-**Current State**: Some tests are marked to be run serially which works for now but makes test time very long.
+            **Goal**: Fix test parallelism issues (mainly in terminal_tools)
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)